datacite/schema45.py

```diff
@@ -39,7 +40,7 @@
 }
 
 validator = validator_factory(
-    pkg_resources.resource_filename("datacite", "schemas/datacite-v4.5.json")
+    str(importlib.resources.files("datacite") / "schemas/datacite-v4.5.json")
```

Comment:
I don't think you need the `str(`

If you want to be consistent with the other versions, you can use `import importlib.resources as importlib_resources` and then `importlib_resources.files("datacite") /`